cmd/pw_ws_broker/main.go

```diff
@@ -139,20 +140,6 @@ func run(c *cli.Context) error {
 		return c.Set("http-addr", ":443")
 	}
 
-	var hasRabbit bool
-	var hasNats bool
-	var hasRedis bool
-	for _, v := range c.FlagNames() {
```

Comment:
The reason i was doing this was to detect if a flag had been specified.

Why? because I was being lazy and using default values.

Without changing the flag definitions and removing the default values you will always get all 3 flags below as true

Reply:
Right you are. Didn't notice that the defaults were populating values. Here's another crack that takes that in to account